planetscale/database_resource.go

```diff
@@ -206,3 +209,57 @@ func (r *databaseResource) Configure(_ context.Context, req resource.ConfigureRe
 
 	r.client = req.ProviderData.(*planetscale.Client)
 }
+
+func splitDatabaseResourceID(id string) (teamID, _id string, ok bool) {
+	attributes := strings.Split(id, "/")
+	requiredAttributesLength := 2
+	if len(attributes) != requiredAttributesLength {
+		return "", "", false
+	}
+	return attributes[0], attributes[1], true
+}
+
+func (r *databaseResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	organizationName, databaseName, ok := splitDatabaseResourceID(req.ID)
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Error importing database",
+			fmt.Sprintf("Invalid id '%s' specified. should be in format \"organization_name/database_name\"", req.ID),
```

Comment:
Since we're not really importing by ID but using an org/db_name combo, would it make sense to return error logs referencing an "ID"? In terms of implementation that's perfectly fine, I'm just worries it might become confusing in the future. 

Same comment would apply in the other two resources. 

Thoughts?

Reply:
Maybe `invalid input` would be better here? What are your thoughts in it?

Reply:
Sounds good to me!